app/views/docs/storage.phtml

```diff
@@ -0,0 +1,83 @@
+<p>Appwrite storage allow you to manage files in your project. It provides APIs to upload, download, delete, and list files. Not only that Appwrite storage service provides APIs to manipulate image. Using the prveview endpoint you can crop, resize, rotate, add borders and border radius and select the desired output format for your image.</p>
+
+<p>Appwrite storage service is a simple and easy to use service. You can use it to store images, videos, documents, and other files for your projects.</p>
+
+<p>With version 0.13 of Appwrite, we also have introduced storage buckets as a container for your files so that you can give more context to your files. Storage buckets is similar to Collections we have in our <a href="/docs/database">Database</a> service. But a bucket gives you more control over what types of files you allow, what is the maximum file size you allow, whether or not to encrypt the files.</p>
+
+<h2>Buckets</h2>
+
+<p> Storage buckets are way to group files into a meaningful collection and give them context. Buckets also provide more power to decide what kinds of files, what sizes you want to allow in that bucket. Whether or not to encrypt the files, scan with antivirus and more. Let us look at how we can create a bucket and configure it to our needs.</p>
+
+<h3>Create Bucket</h3>
+
+<p>You can create your bucket from your Appwrite project's dashboard or using any of our server side SDKs or directly using our REST API authenticated using  an API Key. In order to create a bucket from dashboard, access your Storage from your project's left navigation panel. Click <b>Add Bucket</b> button and choose your bucket's name. For convenience, you can also set a custom ID for your bucket instead of auto-generated ID. This will create a new bucket and take you to it's settings page, where you can configure various options for your bucket.</p>
```

Comment:
```suggestion
<p>You can create your bucket from your Appwrite project's dashboard, using any of our server side SDKs or directly using our REST API authenticated using an API Key. In order to create a bucket from the dashboard, access your Storage from your project's left navigation panel. Click <b>Add Bucket</b> button and choose your bucket's name. For convenience, you can also set a custom ID for your bucket instead of auto-generated ID. This will create a new bucket and take you to its settings page, where you can configure various options for your bucket.</p>
```